### PR TITLE
Suggest in the documentation to wait until the MetalLB controller pod is ready

### PR DESCRIPTION
**Pull request by mdurand54**
Linked to #2971

### CODE REVIEW

File: site/content/docs/user/loadbalancer.md [hunk 1]
```diff
@@ -35,10 +35,13 @@ is not working anymore.
 kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.13.7/config/manifests/metallb-native.yaml
 {{< /codeFromInline >}}
 
-Wait for metallb pods to have a status of Running
+Wait until MetalLB is ready:
```
Comment (aojea) [inline site/content/docs/user/loadbalancer.md:38]: can we be more specific? is the opd with the metallb controller, no?

**Comment by aojea** [comment]
/ok-to-test
/assign @fedepaol 

please sign the CLA 😄

File: site/content/docs/user/loadbalancer.md (hunk 1 above)
Comment (fedepaol) [reply site/content/docs/user/loadbalancer.md]: makes sense to me

File: site/content/docs/user/loadbalancer.md [hunk 2]
```diff
@@ -24,32 +24,35 @@ description: |-
 
 ---
 
-## Installing metallb using default manifests
+## Installing MetalLB using default manifests
```
Comment (BenTheElder) [inline site/content/docs/user/loadbalancer.md:27]: I think this is a breaking change to the heading id for incoming links, but that seems fine this time

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
thank you both!

**Pull request merged**